learning/tour-of-beam/learning-content/io/big-query-io/read-table/java-example/Task.java

```diff
@@ -33,38 +35,42 @@
 import org.apache.beam.sdk.io.gcp.bigquery.BigQueryOptions;
 import org.apache.beam.sdk.options.PipelineOptions;
 import org.apache.beam.sdk.options.PipelineOptionsFactory;
-import org.apache.beam.sdk.transforms.DoFn;
-import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.transforms.*;
```

Comment:
To keep things clean I would recommend not using `.*` instead specific packages. 
May be someone from from java side can tell @Abacn

Reply:
Yes, beam code style forbids `.*` import: https://github.com/apache/beam/blob/master/sdks/java/build-tools/src/main/resources/beam/checkstyle.xml#L82